src/v/kafka/client/transport.h

```diff
@@ -139,6 +139,10 @@ class transport : public rpc::base_transport {
             return dispatch(std::move(r), api_version(2));
         } else if constexpr (std::is_same_v<type, create_topics_request>) {
             return dispatch(std::move(r), api_version(4));
+        } else if constexpr (std::is_same_v<type, sasl_handshake_request>) {
+            return dispatch(std::move(r), api_version(1));
```

Comment:
very cool! I suppose *all* sassl are like this?

Reply:
yes